packages/pillarbox-playlist/src/pillarbox-playlist.js

```diff
@@ -78,13 +97,20 @@ class PillarboxPlaylist extends Plugin {
    */
   constructor(player, options) {
     super(player);
+
     if (options.playlist && options.playlist.length) {
       player.ready(() => {
         this.load(...options.playlist);
       });
     }
+
     this.autoadvance = !!options.autoadvance;
     this.repeat = !!options.repeat;
+    this.previousNavigationThreshold =
+      (typeof options === 'object' && 'previousNavigationThreshold' in options) ?
+      options.previousNavigationThreshold :
+      this.previousNavigationThreshold;
+
```

Comment:
with `options` set to `{}` we could do `Number.isFinite(options.previousNavigationThreshold)` to avoid complex logic

Reply:
Done! Indeed this is much better.